Selenium_Scrapper/selenium_scrape.py

```diff
@@ -59,7 +50,7 @@ def write(self,data):
         self.cursor.executemany(postgres_insert_query, data)
         self.connection.commit()
         count = self.cursor.rowcount
-        print (count, "Record inserted successfully into mobile table")
+        print (count, "Record inserted successfully into table")
 
-a = crawler('/Users/da/Documents/GitHub/auto_crawler/Mock_Data/Nasdaq_tickers.csv','da','','localhost','5432','da')
-a.crawler(0,1)
\ No newline at end of file
+a = crawler('/Users/da/Documents/GitHub/pocket_stock/Mock_Data/Nasdaq_tickers.csv')
```

Comment:
file path should be relative

Reply:
Test script, will be automated with bash script later